### PR TITLE
Add an example AppData file so that Cura is visible in Linux software…

### PR DESCRIPTION
**Pull request by hughsie**
… centers

Fixes: https://github.com/Ultimaker/Cura/issues/530

### CODE REVIEW

File: cura.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      <li>Experts are able to customize 200 settings to achieve the best results</li>
+      <li>Optimized profiles for Ultimaker materials</li>
+      <li>Supported by a global network of Ultimaker certified service partners</li>
+      <li>Print multiple objects at once with different settings for each object</li>
+      <li>Cura supports STL, 3MF and OBJ file formats</li>
+      <li>Open source and completely free</li>
+    </ul>
+  </description>
+  <screenshots>
+    <screenshot type="default" width="1200" height="675">https://people.freedesktop.org/~hughsient/temp/appdata-cura.png</screenshot>
```
Comment (fieldOfView) [inline cura.appdata.xml]: We should probably host a more up to date screenshot somewhere on software.ultimaker.com

Comment (Ghostkeeper) [inline cura.appdata.xml]: You can take this screenshot: http://software.ultimaker.com/Cura.png

Comment (hughsie) [inline cura.appdata.xml]: Update PR, thanks.

**Comment by awhiemstra** [comment]
Note that this breaks the install target because it tries to install the appdata file from CMAKE_BINARY_DIR whereas it is located in CMAKE_SOURCE_DIR

**Pull request merged**